spec/models/data_dot_gov_credentials_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe DataDotGovCredentials do
+  describe '::api_key' do
+    let(:api_key) { DataDotGovCredentials.api_key(force_vcap: true) }
+    it 'returns a string' do
+      expect(api_key).to be_a(String)
```

Comment:
can we update this to look for the exact string in `spec/support/vcap_services.json` ?